spec/spec_helper_acceptance_local.rb

```diff
@@ -34,9 +34,12 @@ def print_parsing_errors
   end
   c.before :suite do
     # Make sure selinux is disabled so the tests work.
-    LitmusHelper.instance.run_shell('setenforce 0', expect_failures: true) if %r{redhat|oracle}.match?(os[:family])
+    LitmusHelper.instance.run_shell('setenforce 0', expect_failures: true) if %r{redhat}.match?(os[:family])
 
     LitmusHelper.instance.run_shell('puppet module install stahnma/epel')
```

Comment:
Unrelated, but given this is here, should this be removed?
https://github.com/puppetlabs/puppetlabs-apache/blob/aabca02e26c34da02fe4d979aaaf31117858ce10/spec/acceptance/mod_authnz_ldap_spec.rb#L9-L14
https://github.com/puppetlabs/puppetlabs-apache/blob/aabca02e26c34da02fe4d979aaaf31117858ce10/spec/acceptance/mod_authnz_ldap_spec.rb#L30-L32

Reply:
no worries. Thanks @ekohl as always. Testing the new changes now. Hopefully it should now work on all OS. If success will push the changes.